src/crystal/system/mime.cr

```diff
@@ -0,0 +1,12 @@
+module Crystal::System::MIME
+  # Load MIME types from operating system source.
+  # def self.load
+end
+
+{% if flag?(:unix) %}
+  require "./unix/mime"
+{% elsif flag?(:win32) %}
```

Comment:
If you're not planning on filling out the windows stub in this PR you should make this raise.

Reply:
It should work on windows quite fine. There just are no OS-supplied mime types available, that's the same as if you have an empty `/etc/mime.types` (etc.) on a Unix system.